README.md

```diff
@@ -1,44 +1,48 @@
-# Vencord Virtmic
+<div align="center">
 
-A simple command line program to allow electron apps to stream with audio on Linux via pipewire
+<img src="https://avatars.githubusercontent.com/u/113042587" width="150">
 
-## Usage
+<br/>
 
-Grab the [prebuild](/prebuilt/) or [build from source](#compiling)
+venmic - screenshare support for pipewire
 
-#### List available targets (apps that are playing audio)
+</div>
 
-This will print a list of names, newline separated
+# 📖 Usage
```

Comment:
you're misusing headings here

`#` is the primary heading (h1), of which there should be exactly 1, which is the title
for each nested header you should increment the level by one, so usage should be ##

Reply:
As the title is beneath the logo I've made all headings h2